### ISSUE TITLE
incomplete stack command on tutorial1

### ISSUE THREAD

**Issue by asheshambasta** [tathougies/beam#264]
The suggested stack command [on this page](http://tathougies.github.io/beam/tutorials/tutorial1/) to start up the repl is:
```
$ stack repl --package beam-core --package beam-sqlite --package sqlite-simple --package beam-migrate
```

This however fails when doing 
```
λ> import Data.Text (Text)

<no location info>: error:
    Could not find module ‘Data.Text’
    It is a member of the hidden package ‘text-1.2.2.2@text-1.2.2.2-9UQZjEJZQFSGMffj1Z5g00’.
``` 

The command to start the repl must instead include the text package; eg a trailing ``` --package text```

**Comment by jappeace** [comment]
you need to add `text` to your `package.yaml` dependencies. This is not a beam issue.

**Comment by asheshambasta** [comment]
@jappeace I know – I'm just saying that the documentation page has a stack command that starts without `text` as a dependency.

**Comment by jappeace** [comment]
I see the problem, I think this patches the docs: https://github.com/tathougies/beam/pull/266

**Comment by jappeace** [comment]
I just don't have the power to merge

**Comment by asheshambasta** [comment]
Thanks.

**Comment by tathougies** [comment]
Fixed in #261
